jujugui/static/gui/src/app/d3-components/d3-components.js

```diff
@@ -162,25 +162,25 @@ YUI.add('d3-components', function(Y) {
     _normalizeHandler: function(handler, module, selector) {
       var result = {};
 
-      if (L.isString(handler)) {
+      if (typeof(handler) === 'string') {
         result.callback = module[handler];
         result.phase = 'on';
       }
 
-      if (L.isObject(handler)) {
+      if (typeof(handler) === 'object' && handler !== null) {
```

Comment:
if handler is an array it'll also return `object`. While that's pretty unlikely, it might be worth a check here as well.